lib/mocha/invocation.rb

```diff
@@ -51,18 +51,18 @@ def threw(tag, value)
       @result = ThrownObject.new(tag, value)
     end
 
-    def arguments
-      @arguments.dup
+    def parameters
+      @parameters
     end
 
     def call_description
-      description = "#{@mock.mocha_inspect}.#{@method_name}#{ParametersMatcher.new(@arguments).mocha_inspect}"
+      description = "#{@mock.mocha_inspect}.#{@method_name}#{ParametersMatcher.new(@parameters.to_a).mocha_inspect}"
```

Comment:
Invocations should only have actual parameters, not matchers, so it seems odd (and makes this change trickier) that `ParametersMatcher` is needed to print their description. Perhaps we can decouple the printing of actual parameters?